cmd/template/cluster/runner.go

```diff
@@ -71,6 +71,7 @@ func (r *runner) run(ctx context.Context, cmd *cobra.Command, args []string) err
 		MasterAZ:          r.flag.MasterAZ,
 		Name:              r.flag.Name,
 		PodsCIDR:          r.flag.PodsCIDR,
+		ExternalSNAT:      r.flag.ExternalSNAT,
```

Comment:
Can we sort alphabetically please? Experience showed this is easier to work with for humans.

Reply:
done